README.md

```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
 - BlackBerry 10
 - Windows Phone 7 and 8
 - Windows (Windows Phone 8.1 only)
-- Tizen
 - Firefox OS
-- Browser
+- Browsers
+    - Chrome
+    - Firefox
+    - Opera
 
-### Android and Amazon Fire OS Quirks
+### Quirks: Android &amp; Amazon Fire OS
 
-- Warning: the Android + Fire OS implementations are greedy and prolonged use will drain the user's battery. 
+**Warning**: the Android and Fire OS implementations are greedy and prolonged use will drain the device's battery.
 
-### Windows Phone 7 and 8 Quirks
+### Quirks: Windows Phone 7 &amp; Windows Phone 8
 
-Windows Phone 7 does not provide native APIs to determine battery
-level, so the `level` property is unavailable.  The `isPlugged`
-parameter _is_ supported.
+The `level` property is _not_ supported on Windows Phone 7 because the OS does not provide native APIs to determine battery level. The `isPlugged` parameter _is_ supported.
 
-### Windows Quirks
+### Quirks: Windows &amp; Windows Phone 8.1
```

Comment:
Scratch that - I see what you have done - I'm OK with it. Let's get this in. It LGTM.